client/ctclient/cmd/ctclient.go

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// ctclient is a command-line utility for interacting with CT logs.
-package main
+// cmd implements a command-line utility for interacting with CT logs.
```

Comment:
If directories aren't necessary, I'd rather not have them? Navigation is already quite painful in some areas of the codebase, because of the depth of the directory structure...

Reply:
The "standard" Cobra way is described here: https://github.com/spf13/cobra-cli/blob/main/README.md#add-commands-to-a-project.

This is one binary (the `main.go`), and all the subcommands are in the same package in `cmd`.

Also, there are a few technical reasons to not separate the subcommands into multiple packages:

- There are a few helpers used across multiple subcommands (e.g. cert printing funcs). Separating them would need a deeper refactoring.
- The export surface would be bigger: now it's just the `cmd.Execute` function, but with separate packages we would need an exported thing per package, or putting it all in `initernal`.